inference-engine/thirdparty/clDNN/kernel_selector/core/cl_kernels/pooling_gpu_ref.cl

```diff
@@ -91,11 +94,15 @@ KERNEL(pooling_gpu)(
 
     ACCUMULATOR_TYPE result = INIT_VAL;
 
+#ifdef SELECTED_INDICES_TYPE
+    SELECTED_INDICES_TYPE result_idx = 0;
```

Comment:
I think we can use `uint `type for this variable

Reply:
Done.